builtin/common/information_formspecs.lua

```diff
@@ -40,24 +50,34 @@ end
 
 core.after(0, load_mod_command_tree)
 
-local function build_chatcommands_formspec(name)
+local function build_chatcommands_formspec(name, sel, copy)
 	local rows = {}
 	rows[1] = "#FFF,0,Command,Parameters"
 
+	local description = "For more information, click on any entry in the list.\n" ..
+		"Double-click to copy the entry to the chat history."
```

Comment:
`Double-click to copy the command to chat` sounds better IMHO. Also, the word `history` is sure to confuse many.